pywebio/output.py

```diff
@@ -361,7 +361,7 @@ def _get_output_spec(type, scope, position, **other_spec):
     return spec
 
 
-def put_text(*texts, sep=' ', inline=False, scope=None, position=OutputPosition.BOTTOM) -> Output:
+def put_text(*texts: List[Any], sep: str = ' ', inline: bool = False, scope: str = None, position: int = OutputPosition.BOTTOM) -> Output:
```

Comment:
```suggestion
def put_text(*texts: Any, sep: str = ' ', inline: bool = False, scope: str = None, position: int = OutputPosition.BOTTOM) -> Output:
```
For variable positional arguments (*args) and variable keyword arguments (**kw) you only need to specify the expected value for one such argument.
See: https://stackoverflow.com/questions/37031928/type-annotations-for-args-and-kwargs